spring-cloud-consul-core/src/main/java/org/springframework/cloud/consul/ConsulAutoConfiguration.java

```diff
@@ -11,6 +13,7 @@
  */
 @Configuration
 @EnableConfigurationProperties
+@ConditionalOnProperty(name = "consul.enabled", matchIfMissing = true)
```

Comment:
Thanks for the feedback. This was intended to piggyback off the ConsulProperties class. I guess the plan there is to update to `spring.consul` as the prefix?